recipe/meta.yaml

```diff
@@ -11,41 +11,53 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-
-  run:
-    - python >=3.6
-    - cycler
-    - databroker
-    - doct
-    - event-model >=1.10.0
-    - historydict
-    - ipython
-    - lmfit
-    - matplotlib
-    - msgpack-python
-    - msgpack-numpy
-    - numpy
-    - ophyd
-    - pyzmq
-    - super_state_machine
-    - toolz
-    - tqdm
-    - zict
+  number: 1
 
 test:
   imports:
     - bluesky
     - bluesky.plans
     - bluesky.callbacks
 
+outputs:
+- name: bluesky-base
+  build:
+    noarch: python
+    script: python -m pip install . --no-deps -vv
```

Comment:
Why did not the templated version work?

Reply:
Earlier versions of CI weren't able to find that variable, but it's plausible that's because my indentation was malformed. I put the template back in.